src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+public class Intake {
+    private CANSparkMax verticalRollerMotor;
+    private CANSparkMax horizontalRollerMotor;
+    private CANSparkMax deploymentMotor;
+    private double suckSpeed;
+    private double spitSpeed;
+    private double inactiveSpeed;
+  
+    public Intake() {
+      // Initializing motor controllers
+      verticalRollerMotor = new CANSparkMax(0, MotorType.kBrushless);
```

Comment:
The rollers on the intake are belted together and only driven by one motor, so you don't need two intake motors.

Reply:
Isn’t there another motor for the centering thing

Reply:
Why are there 3 motors on the CAD then?
![image](https://user-images.githubusercontent.com/68178572/197250965-9a3eb2a8-ae5b-46a5-8e17-9bef92987014.png)

Reply:
No you're right. My bad